runtime/configurationparser/header.h

```diff
@@ -68,14 +69,15 @@ extern "C" {
   void printSet(FILE *, set *, const char *, const char *, const char *);
   void printList(FILE *, list *, const char *, const char *, const char *);
   void visitChildren(block *subject, FILE *file,
-  void visitConfig(FILE *, block *, const char *), 
-  void visitMap(FILE *, map *, const char *, const char *, const char *), 
-  void visitList(FILE *, list *, const char *, const char *, const char *), 
-  void visitSet(FILE *, set *, const char *, const char *, const char *), 
-  void visitInt(FILE *, mpz_t, const char *),
-  void visitFloat(FILE *, mpfr_t, const char *),
-  void visitBool(FILE *, bool, const char *),
-  void visitMInt(FILE *, void *, const char *));
+      void visitConfig(FILE *, block *, const char *), 
+      void visitMap(FILE *, map *, const char *, const char *, const char *), 
+      void visitList(FILE *, list *, const char *, const char *, const char *), 
+      void visitSet(FILE *, set *, const char *, const char *, const char *), 
+      void visitInt(FILE *, mpz_t, const char *),
+      void visitFloat(FILE *, mpfr_t, const char *),
+      void visitBool(FILE *, bool, const char *),
+      void visitMInt(FILE *, void *, const char *),
+      void visitSeparator(FILE *));
```

Comment:
So this is interesting to me. I used KCHAR (currently defined to char) in the strings runtime lib. Are we sure that we will stick with chars going forward? I suppose, even if we do decided to support unicode, that we'll stick with utf-8 instead of the abominations of utf-16 or utf-32.